pywren/wrenhandler.py

```diff
@@ -174,16 +187,26 @@ def generic_handler(event, context_dict):
         response_status['output_key'] = output_key
         response_status['status_key'] = status_key
 
-        KS = get_key_size(s3_client, s3_bucket, data_key)
-        #logger.info("bucket=", s3_bucket, "key=", data_key,  "status: ", KS, "bytes" )
-        while KS is None:
+        data_key_size = get_key_size(s3_client, s3_bucket, data_key)
+        #logger.info("bucket=", s3_bucket, "key=", data_key,  "status: ", data_key_size, "bytes" )
+        while data_key_size is None:
             logger.warning("WARNING COULD NOT GET FIRST KEY")
 
-            KS = get_key_size(s3_client, s3_bucket, data_key)
+            data_key_size = get_key_size(s3_client, s3_bucket, data_key)
         if not event['use_cached_runtime']:
             subprocess.check_output("rm -Rf {}/*".format(RUNTIME_LOC), shell=True)
-
-
+        func_key_size = get_key_size(s3_client, s3_bucket, func_key)
+
+        free_disk_bytes = free_disk_space("/tmp")
```

Comment:
is `/tmp` hardcoded elsewhere as well or is there some variable we can use here cc @apengwin

Reply:
`/tmp` is hardcoded in current wrenhandler. The portability PR abstracts this out to a generic `TEMP` variable that's set at the beginning, depending on what OS is running

Reply:
It is actually hardcoded in the runtimes as well. Unfortunately there are some absolute path dependencies that continue to be a problem in the way conda is packaged.